apps/vr-tests/src/stories/Avatar.stories.tsx

```diff
@@ -111,66 +82,4 @@ storiesOf('Avatar', module)
   ))
   .addStory('size+active+ring-glow', () => (
     <AvatarList images={examples.image} active="active" activeDisplay="ring-glow" />
-  ))
-  .addStory('customSize+image', () => <AvatarCustomSizeList images={examples.image} />)
-  .addStory('customSize+name+badge', () => (
-    <AvatarCustomSizeList names={examples.name} badge="success" />
-  ))
-  .addStory('customSize+icon+active', () => (
-    <AvatarCustomSizeList icon={<ContactIcon />} active="active" />
-  ))
-  .addStory('tokens', () => (

```

Comment:
These tests should be fixed so they continue to work, rather than removed. It looks like you're removing them because you removed the `tokens` prop? How can the same result be achieved with the new styling system?

Reply:
> It looks like you're removing them because you removed the tokens prop? How can the same result be achieved with the new styling system?

Exactly. Currently it's possible to use only styles overrides, I will restore an example for a hexagonal avatar to showcase it.